src/bin/console/console.cpp

```diff
@@ -374,6 +370,9 @@ int Console::_help(const std::string&) {
   out("  commit                                  - Commit a manually created transaction\n");
   out("  txinfo                                  - Print information on the current transaction\n");
   out("  pwd                                     - Print current working directory\n");
+  out("  load_plugin FILE                        - Load and start plugin specified by filepath FILE. It is stored under its ");  // NOLINT
+  out("                                               name which is the filename without the \"lib\" prefix and without the file extension\n");  // NOLINT
+  out("  unload_plugin NAME                      - Stop and unload plugin NAME (also clears the query cache)\n");
```

Comment:
I agree with the transparency property. On the other hand, I imagine some poor Hyrise user sitting in front of the console and wondering why, after unloading a plugin, some queries take more than a nanosecond because they have to be optimised again.